flintrock/flintrock.py

```diff
@@ -226,6 +226,8 @@ def cli(cli_context, config, provider):
 @click.option('--ec2-ebs-optimized/--no-ec2-ebs-optimized', default=False)
 @click.option('--ec2-instance-initiated-shutdown-behavior', default='stop',
               type=click.Choice(['stop', 'terminate']))
+@click.option('--ec2-user-data', default='',
```

Comment:
Would it be appropriate to use Click's `Path` type here? I think it would help us eliminate the need for `_get_user_data()`.

References:
* http://click.pocoo.org/6/api/#click.Path
* http://click.pocoo.org/6/arguments/#file-path-arguments

Example from Flintrock:
* https://github.com/nchammas/flintrock/blob/7e811104184e1d45e02bf0f4056fbc2d53ba2746/flintrock/flintrock.py#L206-L208